tests/cli/workflow/balsamic/store/conftest.py

```diff
@@ -0,0 +1,248 @@
+"""Fixtures for cli balsamic tests"""
+from datetime import datetime
+
+import pytest
+from cg.apps.hk import HousekeeperAPI
+from cg.store import Store, models
+
+from cg.apps.tb import TrailblazerAPI
+
+
+@pytest.fixture
+def balsamic_store_context(balsamic_store, balsamic_case) -> dict:
+    """context to use in cli"""
+    return {
+        "hk_api": MockHouseKeeper(balsamic_case.internal_id),
+        "db": balsamic_store,
+        "tb_api": MockTB(),
+    }
+
+
+class MockTB(TrailblazerAPI):
+    """Mock of trailblazer """
+
+    def __init__(self):
+        """Mock the init"""
+        pass
+
+    def analyses(
+        self,
+        *,
+        family: str = None,
+        query: str = None,
+        status: str = None,
+        deleted: bool = None,
+        temp: bool = False,
+        before: datetime = None,
+        is_visible: bool = None,
+        workflow=None
+    ):
+        return []
+
+
+class MockHouseKeeper(HousekeeperAPI):
+    """Mock HousekeeperAPI"""
+
+    def __init__(self, bundle_name):
+        self.store = MockHousekeeperStore()
+        self.bundle_name = bundle_name
+        self.bundle_data = None
+
+    def get_files(self, bundle: str, tags: list, version: int = None):
+        """Mock get_files of HousekeeperAPI"""
+        del tags, bundle, version
+        return [MockFile()]
+
+    def add_bundle(self, data: dict):
+
+        if self.bundle_data != data:
+            self.bundle_data = data
+            return MockBundle(data=data, name=self.bundle_name), MockVersion()
+
+        return None
+
+
+class MockHousekeeperStore:
+    """Mock Store of Housekeeper"""
+
+    def __init__(self):
+        self.root_dir = ""
+
+    def add_commit(self, *pargs, **kwargs):
+        pass
+
+
+class MockBundle:
+    """Mock Bundle"""
+
+    def __init__(self, data, name):
+
+        self.name = name
+        self._data = data
+
+
+class MockVersion:
+    """Mock Version"""
+
+    def __init__(self):
+        self.created_at = datetime.now()
+        self.included_at = None
+        self.relative_root_dir = ""
+        self.files = []
+
+
+class MockFile:
+    """Mock File"""
+
+    def __init__(self, path=""):
+        self.path = path
+        self.full_path = path
```

Comment:
Provided class description comments seem largely redundant. Class MockVersion is obviously used to mock a version, etc.

Reply:
yes, I guess I'm coding for the linter :(

Reply:
I prefer doing that to adding Meta data in the file btw